src-java/ping-topology/ping-messaging/src/main/java/org/openkilda/messaging/info/flow/FlowPingResponseUtils.java

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2023 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.info.flow;
+
+import java.util.List;
+
+/**
+ * Utility class for {@link FlowPingResponse}.
+ */
+public final class FlowPingResponseUtils {
+
+    private FlowPingResponseUtils() {}
+
+    /**
+     * Build the ping success field.
+     *
+     * @param error error message
+     * @param subFlows list of {@link SubFlowPingPayload}
+     * @return true if the error string is not null and subflows ping was successful, false otherwise
+     */
+    public static boolean buildPingSuccess(String error, List<SubFlowPingPayload> subFlows) {
```

Comment:
In my opinion, we could use slightly different method name. If I got it right the whole idea of the method is to make sure that ping was successful or not.. so I would use the name isPingSuccessful or isPingSuccess. 
But it is just a suggestion, maybe I am missing something.

Reply:
This has been commented here https://github.com/telstra/open-kilda/pull/5176#discussion_r1180318251